lib/driver.js

```diff
@@ -88,10 +104,11 @@ export class ChromiumDriver extends BaseDriver {
     };
 
     this.cd = new Chromedriver(cdOpts);
-    await this.cd.start(this.getSessionCaps());
+    const cdStartRes = /** @type {ChromiumDriverCaps} */(await this.cd.start(this.getSessionCaps()));
```

Comment:
What type cd.start returns? linter complains about it is currently void

Reply:
this was due to the chromedriver dep. now that the linked PR is merged and published, this is not an issue.